database/config.js

```diff
@@ -1,16 +1,149 @@
 const faker = require('faker');
-const mysql = require('mysql');
-const connection = mysql.createConnection({
-  user: 'root',
-  password: '',
-  database: 'about'
-});
-connection.connect();
+const connection = require('./db.js');
 
+let counter = 0;
 
-for (let i = 0; i < 1; i++) {
+const seed = () => {
+  let average = faker.finance.amount(0, 5, 1);
+  let location = faker.finance.amount(0, 5, 1);
+  let cleanliness = faker.finance.amount(0, 5, 1);
+  let service = faker.finance.amount(0, 5, 1);
+  let value = faker.finance.amount(0, 5, 1);
 
+  let certOfExcellence = faker.random.number(1);
+  let greenLeaders = faker.random.number(1);
 
+  let aboutCopy = faker.lorem.paragraphs(5, '\n');
 
+  let amenityIcon1 = faker.random.number(10);
+  let amenityCopy1 = faker.lorem.sentence(2);
+  let amenityIcon2 = faker.random.number(10);
```

Comment:
Do the copies represent the text that shows on a mouseover?

Reply:
great question! i plan on addressing the micro-interactions once i get more structure on the frontend. i'll just reseed with added data at that time.

Reply:
i ended up asking the same thing... I need to be doing that here though